compiler/rustc_middle/src/mir/interpret/value.rs

```diff
@@ -78,6 +78,7 @@ impl<'tcx> ConstValue<'tcx> {
         Some(self.try_to_scalar()?.assert_int())
     }
 
+    #[inline(always)]
```

Comment:
we is that needed?

Reply:
That was based on this suggestion: https://github.com/rust-lang/rust/pull/97019#issuecomment-1131517394